internal/api/v1/deploy/deploy.go

```diff
@@ -0,0 +1,192 @@
+package deploy
```

Comment:
hm, this package is straight forward, but a [package comment](https://github.com/golang/go/wiki/CodeReviewComments#package-comments) could be added.

Reply:
Done, commit e713fc21cfce6b2f7905a1577a64640418d41899